src/IrcClient/Channel.cs

```diff
@@ -41,9 +42,9 @@ public class Channel
     {
         private string           _Name;
         private string           _Key       = String.Empty;
-        private Hashtable        _Users     = Hashtable.Synchronized(new Hashtable(new CaseInsensitiveHashCodeProvider(), new CaseInsensitiveComparer()));
-        private Hashtable        _Ops       = Hashtable.Synchronized(new Hashtable(new CaseInsensitiveHashCodeProvider(), new CaseInsensitiveComparer()));
-        private Hashtable        _Voices    = Hashtable.Synchronized(new Hashtable(new CaseInsensitiveHashCodeProvider(), new CaseInsensitiveComparer()));
+        private ConcurrentDictionary<string, ChannelUser> _Users = new ConcurrentDictionary<string, ChannelUser>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Hmmm, ConcurrentDictionary was added only in .NET 4.0, not sure I want bump the minimum .NET version of SmartIrc4net/Smuxi to 4.0...